pom.xml

```diff
@@ -77,36 +77,6 @@
       <version>1.0.1</version>
       <scope>test</scope>
     </dependency>
-    <dependency>
-    <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
-      <version>1.9.5</version>
-      <scope>test</scope>
```

Comment:
Hmm looks like we're still using these dependencies; was this removal intentional?

Reply:
When I merged the main branch with this branch, some of the dependencies ended up getting duplicated, so I removed the duplicated ones here.